woocommerce/woocommerce-admin
# Remove d3-array dependency

In #1491 we removed `d3-array` dependency, but it was still in package.json. This PR removes it from there too.

### Detailed test instructions:
- Run `rm -rf ./node_modules/` and `npm i`.
- Verify there are no regressions and charts still render.
